Marlin/src/gcode/calibrate/G76_M192_M871.cpp

```diff
@@ -103,9 +103,9 @@ void GcodeSuite::G76() {
     return (timeout && ELAPSED(ms, timeout));
   };
 
-  auto wait_for_temps = [&](const float tb, const float tp, millis_t &ntr, const millis_t timeout=0) {
+  auto wait_for_temps = [&](const celsius_t tb, const celsius_t tp, millis_t &ntr, const millis_t timeout=0) {
     say_waiting_for(); SERIAL_ECHOLNPGM("bed and probe temperature.");
-    while (fabs(thermalManager.degBed() - tb) > 0.1f || thermalManager.degProbe() > tp)
+    while (thermalManager.degBed() != tb || thermalManager.degProbe() > tp)
```

Comment:
hum... this is not exactly the same... I never used this code, but it will give a silly different behaviour than before.